tasks/setup.sh

```diff
@@ -20,6 +20,8 @@
 ####################################################################
 ## Utilities.
 
+ROOT=$( (cd "$(dirname "$0")/.." || exit 1 >/dev/null; pwd -P) )
```

Comment:
Did the earlier relative paths fail with anaconda? Do you envision global use for this or would it be better in install_python_deps_osx()?

Reply:
Would like to leave it there in the hope that we can use it in the future, I think if we update `setup.sh` in the future we can propagate it's use.  With this (if used everywhere) we'd only need to update one line if the script moves, right now we'd need to update a bunch of relative paths if the script moves within the repo.